.travis.yml

```diff
@@ -1,6 +1,4 @@
 language: node_js
-node_js:
-  - "6.1"
 
-notifications:
-  email: false
+node_js:
+  - "8"
```

Comment:
maybe if it is working with 6.1 it should stay this way ?

Reply:
it is easier to officially support just one LTS version